terrawrap/utils/cli.py

```diff
@@ -168,22 +187,29 @@ def _get_retriable_errors(out: List[str]) -> List[str]:
     ]
 
 
-def _post_to_audit_api_url(audit_api_url: str, path: str, exit_code: int, stdout: List[str]):
+def _post_audit_info(
+        audit_api_url: str = None,
+        path: str = None,
+        exit_code: int = None,
+        stdout: List[str] = None,
+        timestamp: int = None,
+        update: bool = False
+):
     root = get_git_root(path)
     path = path.replace(root, '')
 
-    status = 'SUCCESS' if exit_code == 0 else 'FAILED'
-
+    status = 'IN PROGRESS' if not exit_code else ('SUCCESS' if exit_code == 0 else 'FAILED')
```

Comment:
i know you're inheriting this, but it'd be nice to have these vals as constants/enums somewhere